core/common/src/main/java/alluxio/exception/status/ResourceExhaustedRuntimeException.java

```diff
@@ -22,14 +22,24 @@
  */
 public class ResourceExhaustedRuntimeException extends AlluxioRuntimeException {
   private static final Status STATUS = Status.RESOURCE_EXHAUSTED;
-  private static final boolean RETRYABLE = true;
   private static final ErrorType ERROR_TYPE = ErrorType.Internal;
 
   /**
    * Constructor.
    * @param message error message
+   * @param retryable whether it's retryable
    */
-  public ResourceExhaustedRuntimeException(String message) {
-    super(STATUS, message, ERROR_TYPE, RETRYABLE);
+  public ResourceExhaustedRuntimeException(String message, boolean retryable) {
```

Comment:
Seems like all callers called with retryable = false so you can probably remove this parameter. Though, isn't this supposed to be retryable?

Reply:
I think the operation is not retryable since it needs a manual process to scale up the worker. I believe there are other cases that are retryable so keep the signature like this.